exercises/triangle/Triangle.cs

```diff
@@ -9,7 +9,22 @@ public enum TriangleKind
 
 public static class Triangle
 {
-    public static TriangleKind Kind(decimal side1, decimal side2, decimal side3)
+    public static bool IsScalene(double side1, double side2, double side3)
```

Comment:
Just to be consistent, we should keep the types used here and in `Example.cs` the same.